buildspec/linuxIntegrationTests.yml

```diff
@@ -15,13 +15,13 @@ phases:
       - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay&
       - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
       - apt-get update
-      - apt-get install -y jq python2.7 python3.6 python3.7 msbuild
+      - apt-get install -y jq python2.7 python-pip python3.6 python3.7 msbuild
       - aws sts assume-role --role-arn $ASSUME_ROLE_ARN --role-session-name integ-test > creds.json
       - export KEY_ID=`jq -r '.Credentials.AccessKeyId' creds.json`
       - export SECRET=`jq -r '.Credentials.SecretAccessKey' creds.json`
       - export TOKEN=`jq -r '.Credentials.SessionToken' creds.json`
-      - pip install --user aws-sam-cli
-      - pip install --upgrade awscli
+      - pip3 install --user aws-sam-cli
+      - pip3 install --upgrade awscli
```

Comment:
here again I would suggestion using the explicit python version that is intended, with `-m`. Example:

    python3.6 -m pip install --upgrade awscli

Maybe then installing `python-pip` is not be necessary. Either way the explicitness might be useful.